src/Nethermind/Nethermind.Network/Rlpx/RlpxHost.cs

```diff
@@ -78,8 +80,8 @@ public class RlpxHost : IRlpxHost
             //     new[] { new ConsoleLoggerProvider(optionsMonitor) },
             //     new LoggerFilterOptions { MinLevel = Microsoft.Extensions.Logging.LogLevel.Warning });
             // InternalLoggerFactory.DefaultFactory = loggerFactory;
-            
-            _group = new SingleThreadEventLoop();
+
+            _group = new MultithreadEventLoopGroup();
```

Comment:
One thing that we should consider. If we do more writes during the old bodies/receipts sync, we will have more problems with attestations during that time. On the other hand, we will end up the sync faster, so not sure what is better.

Reply:
We can specify number of thread. Default is core count * 2.

Reply:
`Environment.ProcessorCount- 2` ?

Reply:
We can also just ask user to set less peer count if sync is too resource intensive. They would have the same problem if they set very high sync peer count anyway.